package.json

```diff
@@ -45,7 +45,7 @@
     "rollup-bindings": "rollup build/print-bindings.js --file dist/print-bindings.js --format cjs",
     "rollup-benchmark-browser": "rollup benchmark/browser.js --file dist/benchmark.browser.js --format=umd --external=benchmark --globals=benchmark:Benchmark",
     "rollup-benchmark-node": "rollup benchmark/node.js --file dist/benchmark.node.js --format=cjs --external=benchmark",
-    "docker-boot": "docker run -dit --name emscripten -v $(pwd):/src trzeci/emscripten:sdk-tag-1.38.43-64bit bash",
+    "docker-boot": "docker run -dit --name emscripten -v $(pwd):/src:Z trzeci/emscripten:sdk-tag-1.38.43-64bit bash",
```

Comment:
This change is only necessary for Fedora and the few other Linux distros that use SELinux. It *shouldn't* cause a problem otherwise, though, I believe?